homeworks/roman.lytvynenko_Lytvynenko18/GamesProject/Style/style_games.css

```diff
@@ -0,0 +1,89 @@
+.about-me__header {
+  display: grid;
+  grid-template-columns: 10fr 1fr 1fr 1fr;
+  align-content: center;
+  background-color: #EF4934;
+  width: 1280px;
+  height: 80px;
+  font-family: Roboto;
+  text-decoration: none;
+}
+
```

Comment:
Please correct your page according to layout
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/62fb6853-c747-4e44-9022-f67390863de8)
- font size of h1
- padding, the header of the card
- gaps between rows and margin after image
- the same fixed `width` and `height` of the card

Reply:
fixed in 5015c401e868e8132d82c925762abaffb9ef6e48